### PR TITLE
EZEE-2647: Move existing workflows to demo installation

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZEE-2647
> JIRA: https://jira.ez.no/browse/DEMO-266

### CODE REVIEW

File: app/config/workflows.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            workflows:
+                article_workflow:
+                    name: 'Article Workflow'
+                    matchers:
+                        content_type: article
+                        content_status: draft
+                    stages:
+                        draft:
+                            label: 'Draft'
+                            color: '#aaa'
```
Comment (SylvainGuittard) [inline app/config/workflows.yml]:
```suggestion
                            color: '#4a69bd'
```

File: app/config/workflows.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                        design:
+                            label: 'Design'
+                            color: '#9ff'
```
Comment (SylvainGuittard) [inline app/config/workflows.yml]:
```suggestion
                            color: '#cf3d84'
```

File: app/config/workflows.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                        proofread:
+                            label: 'Proofread'
+                            color: '#99f'
```
Comment (SylvainGuittard) [inline app/config/workflows.yml]:
```suggestion
                            color: '#079992'
```

**Comment by SylvainGuittard** [comment]
@webhdx thanks for the PR. I suggested changes for the colors according to the following ticket
https://jira.ez.no/browse/DEMO-266

**Comment by webhdx** [comment]
Thanks @SylvainGuittard added your commits.

**Pull request merged**